pkg/test/resourcefixture/contexts/kms_context.go

```diff
@@ -29,4 +29,15 @@ func init() {
 		//   has value DESTROY_SCHEDULED in field crypto_key_version.state., failedPrecondition
 		SkipDriftDetection: true,
 	}
+	resourceContextMap["kmsautokeyconfig"] = ResourceContext{
+		ResourceKind: "KMSAutokeyConfig",
+		// TestCreateNoChangeUpdateDelete/basic-kmscryptokey: dynamic_controller_integration_test.go:149: value
+		//   mismatch for label with key 'key-one': got 'value-two', want 'value-one'
+		// TestCreateNoChangeUpdateDelete/basic-kmscryptokey: dynamic_controller_integration_test.go:282: reconcile
+		//   returned unexpected error: Delete call failed: error deleting resource: googleapi: Error 400: The request
+		//   cannot be fulfilled. Resource projects/cnrm-test-tgooo56g38yqbn3k/locations/us-central1/keyRings/kmscryptokey-i94182u4ku0un1f0nsna/cryptoKeys/kmscryptokey-i94182u4ku0un1f0nsna/cryptoKeyVersions/1
+		//   has value DESTROY_SCHEDULED in field crypto_key_version.state., failedPrecondition
```

Comment:
Can I ask what crypotykeyversion `DESTROY_SCHEDULED` means for users? Is this a LRO DELETE? Will new requests be rejected? I'm trying to understand how it relates to the autokeyconfig.

Reply:
copy paste error.
corrected it, PTAL